s3/pkg/datastore/yig/conf/yig.toml

```diff
@@ -9,4 +9,5 @@ url="default"
 ceph_dir="/etc/ceph/*.conf"
 
 [database]
+db_type="tidb"
 db_url="root:@tcp(tidb:4000)/yig"
```

Comment:
Where do we initiated database yig?

Reply:
@sfzeng there is a script which init the tables in db before yig backend  is registered. this change will in next pr.

Reply:
@hopkings2008 Could you please create an issue for this and comment here, thanks.

Reply:
@sfzeng thanks for your review, i have created the issue 'https://github.com/opensds/multi-cloud/issues/714' to track this problem you asked.